From: Alex Gonzalez <alexg@balena.io>
Date: Thu, 15 Aug 2024 16:13:03 +0200
Subject: [PATCH] iot-gate-imx8: extend the load address for FDT files

The current difference between the kernel load address and the FDT load
address is roughly 62M, which limits the size of the usable initramfs
when bundled with the kernel.

This commits extends the allowed size to 128M.

Signed-off-by: Alex Gonzalez <alexg@balena.io>
---
 include/configs/iot-gate-imx8.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/iot-gate-imx8.h b/include/configs/iot-gate-imx8.h
index 1a6d513150c8..70e4bc6700fc 100644
--- a/include/configs/iot-gate-imx8.h
+++ b/include/configs/iot-gate-imx8.h
@@ -26,7 +26,7 @@
 	"script=boot.scr\0" \
 	"image=Image.gz\0" \
 	"zip_addr=0x70480000\0" \
-	"fdt_addr=0x44360000\0"	\
+	"fdt_addr=0x48480000\0"	\
 	"fdt_high=0xffffffffffffffff\0"	\
 	"initrd_high=0xffffffffffffffff\0" \
 	"root_opt=rootwait rw\0" \
